lib/src/group_changes.dart

```diff
@@ -32,12 +32,36 @@ class MinMax {
 /// sets as keys in a later map, so we use the keys, and can recover the sets.
 Map<String, List<String>> configSets;
 
+class Result {
+  Result(Map<String, dynamic> data)
+      : previous = data['previous_result'],
+        current = data['result'],
+        expected = data['expected'],
+        // Different methods of accessing BigQuery give different types here.
+        matches = data['matches'] || data['matches'] == 'true',
+        key = [data['previous_result'], data['result'], data['expected']]
+            .join(' ');
+
+  final String previous;
+  final String current;
+  final String expected;
+  final String key;
+  final bool matches;
+
+  bool get newFailure => previous == null && !matches;
+  bool get newPass => previous == null && matches;
+  bool operator ==(dynamic other) => other is Result && key == other.key;
```

Comment:
Should this be Object other instead of dynamic?

Reply:
Yes, I think so.

Reply:
Done